rainfall_project.py

```diff
@@ -1 +1,17 @@
-#filler line
+total_rainfall = 0
+
+list_of_months = ["Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Set", "Oct", "Nov", "Dec"]
+
+year = int(input("Enter the number of years: "))
+
+for years in range(year):
+    print(f"Year {years + 1}")
+    for month in range(12):
+        month_data = (float(input(f"Enter the rainfall amount for {list_of_months[month]}: ")))
+        total_rainfall += month_data
+
+average_rainfall = total_rainfall / (int(year)*12)
+
+print("For 12 Months: ")
+print(f"Total rainfall: {total_rainfall} inches")

```

Comment:
Good job using f strings - one thing that could be done to improve it a little more is limiting the number of digits after the decimal is outputted.

The way to do that is:
`print(f"Total rainfall: {total_rainfall:.2f} inches")`

Reply:
this would limit the result to a value like: 6.01 vs 6.007589